dev/com.ibm.websphere.appserver.org.eclipse.persistence-2.6/com.ibm.websphere.appserver.org.eclipse.persistence-2.6.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.persistence-2.6
+singleton=true
+IBM-Process-Types: server, \
+ client
+-features=com.ibm.websphere.appserver.javax.persistence.base-2.1
+-bundles=com.ibm.websphere.appserver.thirdparty.eclipselink; apiJar=false; location:=dev/api/third-party/
+kind=beta
```

Comment:
this needs to be `kind=ga` if it's going to be included in the `jpa-2.1` feature

Reply:
Changed.